### PR TITLE
Sort Custom Filters

### LINKED ISSUE #6334: Ability to sort/rearrange Custom Filters
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

When adding a custom filter, the new filter is always placed at the bottom of the list of available Custom Filters.

### Describe the solution you'd like

Adding the ability to rearrage/sort the Custom Filters list so that similar filters can be grouped together and so newly added filters don't appear at bottom of list by default.

### Describe alternatives you've considered

If changing the order of the Custom Filters isn't feasible, sorting them alphabetically by default could be helpful as the user could group like filters by name.

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
Sorts custom filters alphabetically

#### Todos
- [ ] Tests
- [ ] Wiki Updates

#### Issues Fixed or Closed by this PR

* closes #6334

### CODE REVIEW

**Review by markus101 — CHANGES_REQUESTED** [review]
We also need to sort them in filter menu

**Comment by stevietv** [comment]
> We also need to sort them in filter menu

done! I missed that as every time I checked the list they were already sorted due to the state being sorted (coming from the modal)

I also added a separator here to split standard from custom filters. Helps a little to distinguish why there are two lists.

File: frontend/src/Components/Menu/FilterMenuContent.js [hunk 1]
```diff
@@ -40,18 +40,25 @@ class FilterMenuContent extends Component {
...
-                filterKey={filter.id}
-                selectedFilterKey={selectedFilterKey}
-                onPress={onFilterSelect}
-              >
-                {filter.label}
-              </FilterMenuItem>
-            );
-          })
+          (customFilters.length > 0) &&
+            <MenuItemSeparator />
```
Comment (markus101) [inline frontend/src/Components/Menu/FilterMenuContent.js:44]:
```suggestion
          customFilters.length > 0 ?
            <MenuItemSeparator /> :
            null
```

**Review by markus101 — APPROVED** [review]
Just the suggestion to make it a ternary, but will commit that change and land once CI is happy.

**Pull request merged**